internal/resources/metal/connection/speed.go

```diff
@@ -19,6 +19,7 @@ var (
 		{2 * giga, "2Gbps"},
 		{5 * giga, "5Gbps"},
 		{10 * giga, "10Gbps"},
+		{100 * giga, "100Gbps"},
```

Comment:
I don't love hiding this fix in an unrelated PR but it's so tiny.

(Side note, though, I have a bigger change to remove this list from the provider and just convert arbitrary speed values from int <=> string and let the API decide what's valid or not.)

Reply:
```
=== RUN   TestSpeedConversion
    speed_test.go:26: Expected error converting invalid speed string to uint, got: 100000000000
    speed_test.go:31: Expected error converting invalid speed uint to string, got: 100Gbps
--- FAIL: TestSpeedConversion (0.00s)
```

Reply:
I've opened an issue to follow-up on this: https://github.com/equinix/terraform-provider-equinix/issues/609